genesyscloud/architect_flow/resource_genesyscloud_architect_flow_proxy.go

```diff
@@ -185,3 +297,124 @@ func getAllArchitectFlowsFn(ctx context.Context, p *architectFlowProxy, name str
 
 	return &totalFlows, nil, nil
 }
+
```

Comment:
I would have Q documented these methods and also generate unit tests if you do not have them